src/Client/Updater.php

```diff
@@ -24,17 +23,11 @@
  *
  * @package Tuf\Client
  */
-class Updater
+class Updater implements LoaderInterface
 {
 
     const MAX_ROOT_DOWNLOADS = 1024;
 
-    /**
-     * The maximum number of bytes to download if the remote file size is not
-     * known.
-     */
-    const MAXIMUM_DOWNLOAD_BYTES = 100000;
```

Comment:
This constant makes more sense on Repository, which is responsible for loading metadata limited by size.